app/components/sdg/related_list_selector_component.html.erb

```diff
@@ -12,7 +12,12 @@
     <%= f.text_field :related_sdg_list,
                       class: "input",
                       placeholder: t("sdg.related_list_selector.placeholder"),
-                      label: t("sdg.related_list_selector.hint"),
+                      label: sanitize(t("sdg.related_list_selector.hint",
+                                      link: link_to(t("sdg.related_list_selector.help.text"),
+                                                      sdg_help_path,
+                                                      title: t("shared.target_blank"),
+                                                      target: "_blank")),
+                                     attributes: %w(href title target)),
```

Comment:
This doesn't seem to work :thinking:; the link opens in the same window. Could you add a test case to confirm/refute this behavior?

I think the cause is the call to `sanitize` in `ConsulFormBuilder#custom_label`.

By the way, Pronto reports:

> Style/PercentLiteralDelimiters: `%w`-literals should be delimited by `[` and `]`. (https://rubystyle.guide#percent-literal-braces)